### PR TITLE
Give driver a  map of handler names to client.TargetHandler

### PR DESCRIPTION
**Pull request by davis-haba**
Signed-off-by: davis-haba <davishaba@google.com>

### CODE REVIEW

**Comment by davis-haba** [comment]
@willbeason

File: constraint/pkg/client/drivers/local/local.go
Comment (willbeason) [inline constraint/pkg/client/drivers/local/local.go:82]: Add a comment that this is a map from handler name to the respective handler.

**Review by willbeason — CHANGES_REQUESTED** [review]
Two minor changes, otherwise looks good

File: constraint/pkg/client/drivers/local/args.go
Comment (maxsmythe) [inline constraint/pkg/client/drivers/local/args.go:103]: Merge conflict indicators

File: constraint/pkg/client/drivers/local/args.go
Comment (davis-haba) [reply constraint/pkg/client/drivers/local/args.go:104]: done

File: constraint/pkg/client/drivers/local/local.go
Comment (davis-haba) [reply constraint/pkg/client/drivers/local/local.go]: done

File: constraint/pkg/client/drivers/local/args.go
Comment (davis-haba) [reply constraint/pkg/client/drivers/local/args.go]: Yep, I tried to rebase off [Will's refactor](https://github.com/willbeason/frameworks/commit/7078cd84b1f2f4f99a5b7acc79e000f88d26f214) which fully moves `Handler` into it's own package, but quickly regretted it. Without Will's client test changes, there will already exist a `handler` type in client. We would have to make a bunch of named imports, only to clean them up later once Will's client changes are merged in.

I think the easiest solution for now is to copy the `TargetHandler` file into the new `handler` package, and defer the proper refactor until Will's PR

@willbeason does this sound reasonable?

**Review by willbeason — APPROVED** [review]

File: constraint/pkg/client/client.go
Comment (willbeason) [inline constraint/pkg/client/client.go:14]: remove newline after `clienterrors` import. Ditto others.

File: constraint/pkg/client/drivers/local/local_unit_test.go
Comment (willbeason) [inline constraint/pkg/client/drivers/local/local_unit_test.go]: Rename "errors2" to "clienterrors" for consistency

File: constraint/pkg/client/crds/validate.go
Comment (willbeason) [inline constraint/pkg/client/crds/validate.go:7]: Rename to `clienterrors` for consistency.

**Review by willbeason — APPROVED** [review]
Just minor changes, otherwise LGTM

File: constraint/pkg/client/client.go
Comment (davis-haba) [reply constraint/pkg/client/client.go]: done

File: constraint/pkg/client/crds/validate.go
Comment (davis-haba) [reply constraint/pkg/client/crds/validate.go]: done

**Review by maxsmythe — APPROVED** [review]
LGTM after lint error fixes

**Comment by willbeason** [comment]
You should be able to fix the lint errors with `golangci-lint run --fix`

**Review by ritazh — APPROVED** [review]
lgtm

**Pull request merged**
